src/nvim/version.c

```diff
@@ -2076,6 +2076,7 @@ void list_in_columns(char_u **items, int size, int current)
   // Sacrifice it to fit in one more column if possible.
   int ncol = (int)(Columns + 1) / width;
   int nrow = item_count / ncol + (item_count % ncol ? 1 : 0);
+  int cur_row = 1;
 
   // i counts columns then rows.  idx counts rows then columns.
```

Comment:
The patch has ` "i" counts`. `idx` should be double-quoted for consistency.

Reply:
Thanks for catching!
I'll amend it when merging it locally.

Reply:
> `idx` should be double-quoted for consistency.

Felt a bit uneasy about it, since it is unnecessary diff from Vim, but since it requested linting for `++i` => `i++` then anyway.. :man_shrugging: